datamodel.customer-survey.xml

```diff
@@ -1880,8 +1880,22 @@ EOF
 		{
 			$sBody = $this->Get('email_body');
 		}
-		$sBody .= '<br/><a href="'.$sQuizzUrl.'">'.Dict::S('Survey-notif-linktoquizz').'</a>';
-
+		
+	
+		$aParams = [];
+		$aParams['recipient->object()'] = $oContact;
+		$aParams['quiz_url'] = $sQuizzUrl;
+		$aParams['this->object()'] = $this;
+		
+		// Email body contains a place holder?
+		$bPrependLink = !preg_match('/\$quiz_url\$/', $sBody);
+		
+		$sBody = MetaModel::ApplyParams($sBody, $aParams);
+		
+		if($bPrependLink == true) {
+			$sBody .= '<br/><a href="'.$sQuizzUrl.'">'.Dict::S('Survey-notif-linktoquizz').'</a>';
+		}
```

Comment:
Actually, the link is being appended, not prepended. The variable name can cause confusion..

Reply:
You are absolutely right.